packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -13,6 +13,7 @@ void main() {
   // See https://flutter.io/testing/ for more info.
   enableFlutterDriverExtension();
 {{/withDriverTest}}
+  // Code directly inside of main() is NOT hot reloadable.
```

Comment:
maybe we should provide a link to explain why?

I don't think we have such a link, but could add one later.